CHANGELOG.md

```diff
@@ -50,8 +50,10 @@
    * CHANGED: Reuse sample::get implementation [#3471](https://github.com/valhalla/valhalla/pull/3471)
    * ADDED: Beta support for interacting with the http/bindings/library via serialized and pbf objects respectively [#3464](https://github.com/valhalla/valhalla/pull/3464)
    * CHANGED: Update xcode to 12.4.0 [#3492](https://github.com/valhalla/valhalla/pull/3492)
+   * CHANGED: fix more protobuf unstable 3.x API [#3494](https://github.com/valhalla/valhalla/pull/3494)
    * CHANGED: fix more protobuf unstable 3.x API [#3494](https://github.com/valhalla/valhalla/pull/3494) & [#3501](https://github.com/valhalla/valhalla/pull/3501)
```

Comment:
seems duplicated with the next line that references both prs, do you want two separate or one with both in it. surely you dont want 3 references?

Reply:
🤦 thanks!